### PR TITLE
Improve docs formatting and add website info to README

### PR DESCRIPTION
**Pull request by henrydavidge**
## What changes are proposed in this pull request?
- Make all parameters and default values in function documentation `monospace`
- Fix the docs and notebook for sample QC functions
- Update README to include project info from the website

## How is this patch tested?
- [ ] Unit tests
- [ ] Integration tests
- [x] Manual tests

`make livehtml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+  <br/>
+  <br/>
+  <a href="https://github.com/projectglow/glow/issues">Issues</a>
+  ·
+  <a href="https://groups.google.com/forum/#!forum/proj-glow">Mailing list</a>
+  ·
+	<a href="https://join.slack.com/t/proj-glow/shared_invite/enQtNzkwNDE4MzMwMTk5LTE2M2JiMjQ1ZDgyYWNkZTFiY2QyYWE0NGI2YWY3ODY3NmEwNmU5OGQzODcxMDBlYzY2YmYzOGM1YTcyYTRhYjA">Slack</a>
+</p>
+
+Glow is an open-source toolkit for working with genomic data at biobank-scale and beyond.
```
Comment (williambrandler) [inline README.md]: Add personas who would use the tool, something like:
Glow is an open-source toolkit to enable bioinformatics and computational biology at biobank-scale and beyond.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+
+# Easy to get started
+If you’ve used Spark before, you don’t need to learn any new APIs to get started with Glow. The
```
Comment (williambrandler) [inline README.md]: don't mention Spark just yet, have an explanation further down

Comment (williambrandler) [reply README.md]: s/If you’ve used Spark before, //

File: README.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+toolkit includes the building blocks that you need to perform the most common analyses right away:
+
+- Datasources for loading VCF and BGEN files into Spark DataFrames
+- Functions for performing quality control and data manipulation
+- Variant normalization and lift over
+- Regression functions
+- Integration with Spark ML libraries for population stratification
```
Comment (williambrandler) [inline README.md]: s/Spark ML/machine learning/

File: README.md [hunk 4]
```diff
@@ -1,3 +1,45 @@
...
+- Utilities for piping DataFrames through command line tools
+
+# Built to scale
+Glow makes genomic data work with Spark SQL, the leading engine for working with large structured
```
Comment (williambrandler) [inline README.md]: s/Spark SQL/Spark/

File: README.md [hunk 5]
```diff
@@ -1,3 +1,45 @@
...
+datasets. It fits natively into the ecosystem of tools that have enabled thousands of organizations
```
Comment (williambrandler) [inline README.md]: Organizations from biotechnology startups to Fortune 100 companies and governments leverage Spark to scale up their data analytics workflows. Glow bridges the gap between bioinformatics and the Spark ecosystem.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,45 @@
...
+to scale their workflows to petabytes of data.
+
+# Flexible
+Glow works with datasets in common file formats like VCF or BGEN as well as common big data
```
Comment (williambrandler) [inline README.md]: VCF, BGEN, or Plink

Comment (williambrandler) [reply README.md]: s/common/high performance/

File: README.md [hunk 7]
```diff
@@ -1,3 +1,45 @@
...
+standards. You can write queries using the native Spark SQL APIs in Python, SQL, R, Java, and Scala.
+The same APIs allow you to bring your genomic data together with other datasets like electronic
```
Comment (williambrandler) [inline README.md]: s/like/such as/

File: README.md [hunk 8]
```diff
@@ -1,3 +1,45 @@
...
+	<a href="https://join.slack.com/t/proj-glow/shared_invite/enQtNzkwNDE4MzMwMTk5LTE2M2JiMjQ1ZDgyYWNkZTFiY2QyYWE0NGI2YWY3ODY3NmEwNmU5OGQzODcxMDBlYzY2YmYzOGM1YTcyYTRhYjA">Slack</a>
+</p>
+
+Glow is an open-source toolkit for working with genomic data at biobank-scale and beyond.
+
+# Easy to get started
+If you’ve used Spark before, you don’t need to learn any new APIs to get started with Glow. The
+toolkit includes the building blocks that you need to perform the most common analyses right away:
+
+- Datasources for loading VCF and BGEN files into Spark DataFrames
```
Comment (williambrandler) [inline README.md]: Load VCF, BGEN and Plink files in a distributed fashion

File: README.md [hunk 9]
```diff
@@ -1,3 +1,45 @@
...
+- Functions for performing quality control and data manipulation
```
Comment (williambrandler) [inline README.md]: Perform common quality control and data manipulation with pre-built functions

File: README.md [hunk 10]
```diff
@@ -1,3 +1,45 @@
...
+- Variant normalization and lift over
+- Regression functions
```
Comment (williambrandler) [inline README.md]: Perform genome-wide association studies

File: README.md (hunk 3 above)
Comment (williambrandler) [reply README.md]: s/Integration/Integrate/ (to fit with the style above)

File: README.md [hunk 11]
```diff
@@ -1,3 +1,45 @@
...
+- Integration with Spark ML libraries for population stratification
+- Utilities for piping DataFrames through command line tools
```
Comment (williambrandler) [inline README.md]: Integrate command line bioinformatics tools with DataFrames, to simplify data processing and analysis

File: README.md (hunk 4 above)
Comment (williambrandler) [reply README.md]: ^most bioinformatics don't use SQL, so don't mention it just yet

File: README.md [hunk 12]
```diff
@@ -1,3 +1,46 @@
...
+
+# Easy to get started
+The toolkit includes the building blocks that you need to perform the most common analyses right away:
+
+- Load VCF, BGEN, and Plink files into distributed DataFrames
+- Perform quality control and data manipulation with built-in functions
+- Variant normalization and lift over
+- Perform genome-wide association studies
+- Integrate with Spark ML libraries for population stratification
+- Parallelize command line tools to scala existing workflows
```
Comment (williambrandler) [inline README.md]: s/to scala/to scale/

File: README.md [hunk 13]
```diff
@@ -1,3 +1,46 @@
...
+</p>
+
+Glow is an open-source toolkit to enable bioinformatics at biobank-scale and beyond.
+
+# Easy to get started
+The toolkit includes the building blocks that you need to perform the most common analyses right away:
+
+- Load VCF, BGEN, and Plink files into distributed DataFrames
+- Perform quality control and data manipulation with built-in functions
+- Variant normalization and lift over
```
Comment (williambrandler) [inline README.md]: s/lift over/liftOver/

File: README.md (hunk 12 above)
Comment (henrydavidge) [reply README.md]: 😱

**Comment by henrydavidge** [comment]
Merging since @williambrandler is traveling

**Pull request merged**